sdk/resourcemover/Azure.ResourceManager.Migrate/api/Azure.ResourceManager.Migrate.netstandard2.0.cs

```diff
@@ -42,32 +42,32 @@ public partial class MoveCollectionResource : Azure.ResourceManager.ArmResource
-        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.Migrate.Models.RequiredForResourcesCollection>> GetRequiredForAsync(string sourceId, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual Azure.Pageable<Azure.ResourceManager.Migrate.Models.UnresolvedDependency> GetUnresolvedDependencies(Azure.ResourceManager.Migrate.Models.DependencyLevel? dependencyLevel = default(Azure.ResourceManager.Migrate.Models.DependencyLevel?), string orderby = null, string filter = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual Azure.AsyncPageable<Azure.ResourceManager.Migrate.Models.UnresolvedDependency> GetUnresolvedDependenciesAsync(Azure.ResourceManager.Migrate.Models.DependencyLevel? dependencyLevel = default(Azure.ResourceManager.Migrate.Models.DependencyLevel?), string orderby = null, string filter = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual Azure.ResourceManager.ArmOperation<Azure.ResourceManager.Migrate.Models.OperationStatus> InitiateMove(Azure.WaitUntil waitUntil, Azure.ResourceManager.Migrate.Models.ResourceMoveContent content = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual System.Threading.Tasks.Task<Azure.ResourceManager.ArmOperation<Azure.ResourceManager.Migrate.Models.OperationStatus>> InitiateMoveAsync(Azure.WaitUntil waitUntil, Azure.ResourceManager.Migrate.Models.ResourceMoveContent content = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual Azure.ResourceManager.ArmOperation<Azure.ResourceManager.Migrate.Models.OperationStatus> Prepare(Azure.WaitUntil waitUntil, Azure.ResourceManager.Migrate.Models.PrepareContent content = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual System.Threading.Tasks.Task<Azure.ResourceManager.ArmOperation<Azure.ResourceManager.Migrate.Models.OperationStatus>> PrepareAsync(Azure.WaitUntil waitUntil, Azure.ResourceManager.Migrate.Models.PrepareContent content = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.Migrate.Models.RequiredForResourcesList> GetRequiredFor(string sourceId, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
The method name is a bit confusing as RequiredFor is not a noun after Get.

Reply:
Renamed to `GetRequiredForResources`